opencti-platform/opencti-front/src/private/components/profile/ProfileOverview.js

```diff
@@ -301,17 +286,13 @@ const ProfileOverviewComponent = (props) => {
 
   return (
     <div>
-      <Dialog
-        open={display2FA}
-        PaperProps={{ elevation: 1 }}
-        keepMounted={false}
-        onClose={() => setDisplay2FA(false)}
-      >
+      <Dialog open={display2FA} PaperProps={{ elevation: 1 }}
+              keepMounted={false} onClose={() => setDisplay2FA(false)}>
         <DialogTitle style={{ textAlign: 'center' }}>
           {t('Enable two-factor authentication')}
         </DialogTitle>
         <DialogContent>
-          <OtpComponent closeFunction={() => setDisplay2FA(false)} />
```

Comment:
Can you recheck you commited file with the 
<img width="742" alt="image" src="https://user-images.githubusercontent.com/50013774/196401096-acbcb479-39ec-4db2-bf5c-5ff0ad0d6fda.png">
empty tag box checked

Reply:
Fixed